### DISCUSSION TITLE
Fair relaunch of CODE dApp and mint process

### DISCUSSION THREAD

**Discussion by axeexcess** [Tradecoin-Token/A4ADAO#11] [Ideas]
📖 A4ADAO_PROPOSAL_1 🔨
Title: Fair relaunch of CODE dApp and mint process

Author: @axeexcess

Overview: We are seeking to propose the fair relaunch of the CODE dApp and minting process. We consider the 1st launch to have taken place in a rushed manner without clear communication to the majority of CODE holders of what the rewards and implications would be for minting late by even 1 block/minute. Case in point, users were not made clearly aware before time that staking their 0.001 share of CODE would result in a 191+ CODE reward at block 1 but users who would be even 10 minutes late to the mint would get less than 10 CODE. At the time of this proposal, 10 users are due for a reward of 191+ CODE while 13 other users will get less than 10 CODE. We submit that the CODE whitepaper needs to be updated with clear examples of what minting at the genesis block entails and what the penalties are for minting late. Information on what the subsequent staking periods will be after the 1st 7-day period should also be published for users so they can plan their stakes accordingly. An Excel sheet with the calculations of rewards would be very welcome. After all these updates and tools are made available for users, an announcement should be made in the A4A Dao Announcement group and users given 24 hours notice before the relaunch of the CODE dApp and minting process. We feel this would give all CODE holders a fair chance and would not generate potential whales within the first day of the project.

Decisions: Vote option: Y / N

    Y: Relaunch CODE dApp and mint process
    N: Leave CODE dApp and mint process as is

Required Quorum: 51%

Required CODE  Participation: 0.001 CODE

 How to vote:

Send a transaction of 0.00001 A4A with the attachment format:

"prop1 Y"
or
"prop1 N" 

to: 3JhE2u6TTeZ9uim3799JxnZ9e9tMUfiCMCA

 Voting will end on Oct 20th

**Comment by Tradecoin-Token** [comment]
**Voting Results were announced on Telegram:** https://telegram.me/A4ADAO/9600

**Voting Results based on Votes Counting**
11- Yes
**18- No**

**Voting Results based on A4A Weightage**
Votes Counting
**119881.67-No (Winner)**

**Based on Weightage as well as Votes Counting prop1 was rejected.**
